release.py

```diff
@@ -1,176 +1,195 @@
-from __future__ import print_function

```

Comment:
Apologies for the messy diff here—there are only additions made to release.py, not removals. GitHub is confused from the Debian merge.